xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentParking_version.xsd

```diff
@@ -127,9 +127,9 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>whether shelter is enclosed.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="Covered" type="xsd:boolean" minOccurs="0">
+			<xsd:element name="Covered" type="CoveredEnumeration" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Whether storage is covered.</xsd:documentation>
+					<xsd:documentation>Whether the component is indoors or outdoors.</xsd:documentation>
```

Comment:
```suggestion
					<xsd:documentation>Whether the parking space is protected from weather.</xsd:documentation>
```

Reply:
Not really a sentence either in my opinion, but at least a full half-sentence now :-) Please feel free adding another half or keeping the initial solution.